Ollama models config incorrect

current values.yaml
```yaml
ollama:
  # -- Automatically install Ollama Helm chart from https://otwld.github.io/ollama-helm/. Use [Helm Values](https://github.com/otwld/ollama-helm/#helm-values) to configure
  enabled: true
  # -- If enabling embedded Ollama, update fullnameOverride to your desired Ollama name value, or else it will use the default ollama.name value from the Ollama chart
  fullnameOverride: "open-webui-ollama"
  # -- Example Ollama configuration with nvidia GPU enabled, automatically downloading a model, and deploying a PVC for model persistence
  # ollama:
  #   gpu:
  #     enabled: true
  #     type: 'nvidia'
  #     number: 1
  #   models:
  #     - llama3
  # runtimeClassName: nvidia
  # persistentVolume:
  #   enabled: true
  #   volumeName: "example-pre-existing-pv-created-by-smb-csi"
```

Helm Error:
```bash
$ helm upgrade --install open-webui open-webui/open-webui --values ollama-webui-values.yaml
coalesce.go:286: warning: cannot overwrite table with non table for open-webui.ollama.ollama.models (map[pull:[] run:[]])
coalesce.go:286: warning: cannot overwrite table with non table for open-webui.ollama.ollama.models (map[pull:[] run:[]])
coalesce.go:286: warning: cannot overwrite table with non table for open-webui.ollama.ollama.models (map[pull:[] run:[]])
Error: UPGRADE FAILED: template: open-webui/charts/ollama/templates/deployment.yaml:148:27: executing "open-webui/charts/ollama/templates/deployment.yaml" at <.Values.ollama.models.pull>: can't evaluate field pull in type interface {}
```

Parameter is `ollama.ollama.models.pull`
see: https://github.com/otwld/ollama-helm/blob/main/values.yaml#L79

correct comment:
```yaml
ollama:
  # -- Automatically install Ollama Helm chart from https://otwld.github.io/ollama-helm/. Use [Helm Values](https://github.com/otwld/ollama-helm/#helm-values) to configure
  enabled: true
  # -- If enabling embedded Ollama, update fullnameOverride to your desired Ollama name value, or else it will use the default ollama.name value from the Ollama chart
  fullnameOverride: "open-webui-ollama"
  # -- Example Ollama configuration with nvidia GPU enabled, automatically downloading a model, and deploying a PVC for model persistence
  # ollama:
  #   gpu:
  #     enabled: true
  #     type: 'nvidia'
  #     number: 1
  #   models:
  #     pull:
  #      - llama3
  # runtimeClassName: nvidia
  # persistentVolume:
  #   enabled: true
  #   volumeName: "example-pre-existing-pv-created-by-smb-csi"
```

--- a/charts/open-webui/Chart.yaml
+++ b/charts/open-webui/Chart.yaml
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: open-webui
-version: 5.10.0
+version: 5.10.1
 appVersion: 0.5.10
 home: https://www.openwebui.com/
 icon: >-

--- a/charts/open-webui/values.yaml
+++ b/charts/open-webui/values.yaml
@@ -13,7 +13,10 @@ ollama:
   #     type: 'nvidia'
   #     number: 1
   #   models:
-  #     - llama3
+  #     pull:
+  #       - llama3
+  #     run:
+  #       - llama3
   # runtimeClassName: nvidia
   # persistentVolume:
   #   enabled: true

